botorch/optim/optimize.py

```diff
@@ -33,7 +33,7 @@ def optimize_acqf(
     bounds: Tensor,
     q: int,
     num_restarts: int,
-    raw_samples: int,
+    raw_samples: Optional[int] = None,
```

Comment:
Shouldn't there be a default value or an error message when `raw_samples` is `None` and `batch_initial_conditions` is not specified? Otherwise, this will lead to a `TypeError` in `gen_batch_initial_conditions`, which is not super user friendly.

Reply:
I think an error message is more appropriate here.